src/components/TokenSelector.jsx

```diff
@@ -6,27 +6,35 @@ import Checkbox from 'material-ui/Checkbox';
 class TokenSelector extends React.Component {
   constructor(props) {
     super(props);
-    this.state = {
```

Comment:
Really, this component could easily be a functional component. It doesn't need local state, and other components will need its state (for highlighting certain tokens, etc). I would move the state up to `AppBody` and pass down the state as a prop